flipperdevices/flipperzero-firmware
# [FL-1486] Fix leaks and leak-sanitizer

# What's new

- Fixed leak sanitizer leaks
- Add heap difference to catch 'false leaks'

# Verification 

- Start empty application, allocate some memory, free it
- Open terminal, check there is additional info for memory deallocation after task termination

# Checklist (do not modify)
